### PR TITLE
New frontend to the webapp

### PR DESCRIPTION
**Pull request by meets2tarun**
#1 Using bootstrap

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Uh verified commits! I need to look how to do that :)

**Comment by niccokunzmann** [comment]
There are conflicts @meets2tarun because of my other merge.

**Comment by abishekvashok** [comment]
@meets2tarun pls rebase.

**Comment by meets2tarun** [comment]
>Uh verified commits! I need to look how to do that :)

 @niccokunzmann You can easily make your commits verified using your gpg keys 
 https://help.github.com/articles/signing-commits-using-gpg/

**Comment by xeon-zolt** [comment]
resolved the conflict
i have changed it to my fork for release as its still under beta and testing will make it look dirty on meilix release page

File: templates/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<!DOCTYPE html>
```
Comment (meets2tarun) [inline templates/index.html]: It's required I think?

**Comment by meets2tarun** [comment]
> resolved the conflict
i have changed it to my fork for release as its still under beta and testing will make it look dirty on meilix release page

@xeon-zolt 
Thank You

**Comment by mariobehling** [comment]
@meets2tarun @xeon-zolt Let's try to get two approvals for PRs in the future and speed up the review process.

**Pull request merged**